amazon_msk/assets/configuration/spec.yaml

```diff
@@ -7,6 +7,14 @@ files:
       - template: init_config/openmetrics_legacy
   - template: instances
     options:
+    - name: use_openmetrics
+      description: |
+        Whether or not to use the latest version of OpenMetrics functionality for more features and performance.
```

Comment:
```suggestion
        Use the latest version of OpenMetrics for more features and better performance.
```

Reply:
Here I'm referring to our OpenMetrics implementation rather than the spec